src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -0,0 +1,43 @@
+import random as _random
+import turicreate as _tc
+from turicreate import extensions as _extensions
+from turicreate.toolkits._model import CustomModel as _CustomModel
+
+def create(dataset, target, feature=None, batch_size=0, max_iterations=0,
+           seed=None, verbose=True):
+    model = _extensions.one_shot_object_detector()
+    if seed is None: seed = _random.randint(0, (1<<31)-1)
```

Comment:
@hoytak Do we have a standard way of randomly seeding that should be factored out? At the least, I think we should copy mechanisms used elsewhere in Turi Create

Reply:
Eh, this is fine (with comment to 64 bit).  Possibly preferable to the other way, which is just time.time().  This punts it to the python seeding system.